doto-frontend/src/helpers/DotoService.js

```diff
@@ -54,6 +54,36 @@ const DotoService = {
             data: newTask,
         });
 
+        // Todo; catch for errors depending if it didn't post properly or maybe retry mechanism
+    },
+    getUserInfo: async () => {
+        const path = baseUrl + "/user/get";
+
+        try {
+            const response = await axios.get(path, {
+                headers: { Authorization: "Bearer " + CookieManager.get("jwt") },
+            });
+            const userInfo = response.data;
+            console.log(userInfo.themePreference);
```

Comment:
Can we get rid of this console.log statements and any others that exist

Don't remove the console.log for errors though